Support bypassing by partitions (#13581)
Alluxio/alluxio: docs/en/operation/User-CLI.md

    * `<UDB_TYPE>`: the UDB type
    * `<UFS_PREFIX>`: the UFS path prefix that the mount properties are for
    * `<MOUNT_PROPERTY>`: an Alluxio mount property
  * `catalog.db.config.file`: the config file for the UDB, 

>     you can configure which tables and partitions should be bypassed from Alluxio by this option.

Reviewer: ```suggestion
    you can configure which tables and partitions to bypass from Alluxio in a configuration specified by this option.
```
